src/apps/api/views/competitions.py

```diff
@@ -72,6 +72,26 @@ def toggle_publish(self, request, pk):
         competition.save()
         return Response({"published": competition.published})
 
+    @action(detail=True, methods=('GET',))
+    def get_files(self, request, pk):
+        competition = Competition.objects.select_related(
+            'created_by'
+        ).prefetch_related(
+            'dumps__dataset',
+        ).get(id=pk)
+        if request.user != competition.created_by and request.user not in competition.collaborators.all():
+            raise PermissionDenied("You don't have access to the competition files")
+        from utils.data import make_url_sassy
+        bundle = competition.bundle_dataset
+        files = {
+            'bundle': {
+                'name': bundle.name,
+                'url': make_url_sassy(bundle.data_file.name)
+            },
+            'dumps': [{'name': dump.dataset.name, 'url': make_url_sassy(dump.dataset.data_file.name)} for dump in competition.dumps.all()]
```

Comment:
indentation here?

Reply:
indent how? it is list comprehension, which I never know how to indent in a way that doesn't look confusing to me.

Reply:
Yeah I misinterpreted what this was doing before, it'd probably be best pulled out as a standalone variable in a boring old loop for clarity sake... but it's fine either way imo, will leave it up to you

```python
dumps = []
for dump in competition.dumps.all()
    dumps.append({'name': d.dataset.name, 'url': make_url_sassy(d.dataset.data_file.name)})
```